applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -606,6 +606,21 @@ The current maximum number of Bluetooth LE connections that can be selected usin
 
                west build -b nrf7002dk/nrf5340/cpuapp -- -DCONFIG_BRIDGED_DEVICE_BT=y -DEXTRA_CONF_FILE="overlay-bt_max_connections_app.conf" -Dipc_radio_EXTRA_CONF_FILE="overlay-bt_max_connections_net.conf"
 
+Configuring Bluetooth LE connection and scan parameters
+-------------------------------------------------------
+
+You can set your own Bluetooth LE connection parameters instead of accepting the default ones requested by the peripheral device. To do this, enable the ``CONFIG_BRIDGE_FORCE_BT_CONNECTION_PARAMS=y`` option, which allows you to modify the parameters.
+
+Once enabled, you can adjust the following Kconfig values to set the desired parameters:
+
+- ``CONFIG_BRIDGE_BT_SCAN_WINDOW`` - Time duration of a central actively scanning for devices within scan interval.
+- ``CONFIG_BRIDGE_BT_SCAN_INTERVAL`` - Time between consecutive Bluetooth scan windows.
+- ``CONFIG_BRIDGE_BT_CONNECTION_INTERVAL_MIN`` - It is the minimum duration of time requested by central(bridge) after the peripheral device should wake up to communicate.
+- ``CONFIG_BRIDGE_BT_CONNECTION_INTERVAL_MAX`` - It is the maximum duration of time requested by central(bridge) after the peripheral device should wake up to communicate.
+- ``CONFIG_BRIDGE_BT_CONNECTION_TIMEOUT`` - The time since the last packet was successfully received until the devices consider the connection lost.
+- ``CONFIG_BRIDGE_BT_CONNECTION_LATENCY`` - It allows the peripheral to skip waking up for a certain number of connection events if it does not have any data to send.
```

Comment:
```suggestion
- :kconfig:option:`CONFIG_BRIDGE_BT_SCAN_WINDOW` - The duration a central actively scans for devices within the scan interval.
- :kconfig:option:`CONFIG_BRIDGE_BT_SCAN_INTERVAL` - Time between consecutive Bluetooth scan windows.
- :kconfig:option:`CONFIG_BRIDGE_BT_CONNECTION_INTERVAL_MIN` - The minimum time requested by the central (the bridge) after which the peripheral device should wake up to communicate.
- :kconfig:option:`CONFIG_BRIDGE_BT_CONNECTION_INTERVAL_MAX` - The maximum time requested by the central (the bridge) after which the peripheral device should wake up to communicate.
- :kconfig:option:`CONFIG_BRIDGE_BT_CONNECTION_TIMEOUT` - The time since the last packet was successfully received until the devices consider the connection lost.
- :konfig:option:`CONFIG_BRIDGE_BT_CONNECTION_LATENCY` - Allows the peripheral to skip waking up for a certain number of connection events if it does not have any data to send.
```